internal/resources/volumes.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	userSecretVolName = "users-config"
+	userSecretVolName = "users-config" // #nosec
```

Comment:
I'm very curious, what what did the tool think was wrong with this line?

Reply:
@phlogistonjohn  As noted in the corresponding rule [g101](https://securego.io/docs/rules/g101.html), "this can be noisy and may generate false positives" (which is indeed, the case here).

Reply:
Oh, it picks up on the term "Secret" that's pretty annoying. But thanks for explaining it.